src/org/usfirst/frc/team6880/robot/task/TaskTurnLeft90deg.java

```diff
@@ -0,0 +1,47 @@
+package org.usfirst.frc.team6880.robot.task;
+
+import org.usfirst.frc.team6880.robot.FRCRobot;
+
+public class TaskTurnLeft90deg implements RobotTask {
+	FRCRobot robot;
+	double endDirection;
+	double currentDirection;
+	boolean wrapNegYaw;
+	
+	public TaskTurnLeft90deg(FRCRobot robot) 
+	{
+		this.robot = robot;
+	}
+	
+	public void initTask()
+	{
+		//Calculate the end direction
+		endDirection = robot.navigation.gyro.getYaw() + 90.0;

```

Comment:
Is the robot spinning clockwise or counter clockwise?  I think +90 will make it end at 90 degrees clockwise from the starting position.